custom_platforms.js

```diff
@@ -236,8 +250,11 @@ module.exports = {
                 "important,protobuf",
                 "\\$extension",
                 ",extension",
-                "\\$(.*,)?hls=",
-                "\\$(.*,)?jsonprune="
+                ",domain=\/",
```

Comment:
Is it intentionally excluded from Chrome/Edge/Opera/Firefox extensions?
If so, shouldn't `\\$domain=\/` be also added?

Reply:
Extensions for Chromium based browsers and FF will support this modifier later.